android/java/AndroidManifest.xml

```diff
@@ -231,6 +231,16 @@
     </intent-filter>
 </service>
 
+<service
+    android:name="org.chromium.chrome.browser.autofill.BraveAutofillService"
+    android:label="@string/app_name"
+    android:permission="android.permission.BIND_AUTOFILL_SERVICE"
+    android:exported="true">
```

Comment:
CC @stoletheminerals as we export the service here.

Reply:
Thanks @deeppandya. This is fine, we have to export and `android.permission.BIND_AUTOFILL_SERVICE` is a system-level permission, so only system apps can bind to this service.